deploy/charts/harvester/templates/configmap.yaml

```diff
@@ -0,0 +1,21 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: harvester-helpers
+  labels:
+{{ include "harvester.labels" . | indent 4 }}
+data:
+  promote.sh: |
+    #!/bin/bash -ex
+    echo start promote
```

Comment:
Is it simpler to package the scripts in the harvester server image? Or we want it to be configurable somehow